package.json

```diff
@@ -54,12 +54,13 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
-	  "is": "3.1.0",
-	  "optimist": "0.6.1",
-	  "npm-package-arg": "4.2.0",
-	  "tar.gz": "1.0.2",
-	  "rimraf": "2.5.2",
-	  "npm": "3.10.3"
+    "fs-extra": "1.0.0",
```

Comment:
In the original, this whole clause used hard tabs for indents, unlike the rest of the file. I converted them to spaces.